zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -829,3 +829,5 @@ def flow(  # pylint:disable=too-many-branches, too-many-statements
             delete_if_local(*dst_temps)
         if self.clear_cache_on_return:
             clear_cache(*op_args, **op_kwargs)
+
+        return tasks
```

Comment:
Does it make sense to return tasks rather than the task's output?

Reply:
Task output is not known at this time